spec-tests/src/test/java/org/candlepin/spec/content/ContentAccessSpecTest.java

```diff
@@ -0,0 +1,864 @@
+        assertEquals(content1.getLabel(), certContent1.get("label").asText());
+        assertEquals(content1.getVendor(), certContent1.get("vendor").asText());
+        assertEquals(content1.getArches(), certContent1.get("arches").get(0).asText());
+        verifyCertContentPath(ownerKey, content1.getContentUrl(), null, certContent1.get("path").asText());
+
+
+        assertEquals(content3.getType(), certContent3.get("type").asText());
+        assertEquals(content3.getName(), certContent3.get("name").asText());
+        assertEquals(content3.getLabel(), certContent3.get("label").asText());
+        assertEquals(content3.getVendor(), certContent3.get("vendor").asText());
+        assertEquals(0, certContent3.get("arches").size());
+        verifyCertContentPath(ownerKey, content3.getContentUrl(), null, certContent3.get("path").asText());
+    }
+
+    @Test
+    public void shouldAllowChangingTheContentAccessModeAndModeList() throws Exception {
+        ApiClient adminClient = ApiClients.admin();
+        OwnerDTO owner = adminClient.owners()
+            .createOwner(Owners.random().contentAccessMode(Owners.SCA_ACCESS_MODE));
+        String ownerKey = owner.getKey();
+
+        owner.contentAccessMode(Owners.ENTITLEMENT_ACCESS_MODE);
+        adminClient.owners().updateOwner(ownerKey, owner);
+        assertThat(adminClient.owners().getOwner(ownerKey))
+            .returns(Owners.ENTITLEMENT_ACCESS_MODE, OwnerDTO::getContentAccessMode)
+            .returns(Owners.ACCESS_MODE_LIST, OwnerDTO::getContentAccessModeList);
+
+        owner.contentAccessModeList(Owners.ENTITLEMENT_ACCESS_MODE);
+        adminClient.owners().updateOwner(ownerKey, owner);
+        assertThat(adminClient.owners().getOwner(ownerKey))
+            .returns(Owners.ENTITLEMENT_ACCESS_MODE, OwnerDTO::getContentAccessMode)
+            .returns(Owners.ENTITLEMENT_ACCESS_MODE, OwnerDTO::getContentAccessModeList);
+    }
+
+    @Test
+    public void shouldAssignTheDefaultModeAndListWhenNoneIsSpecified() throws Exception {
+        ApiClient adminClient = ApiClients.admin();
+        OwnerDTO owner = new OwnerDTO()
+            .id(StringUtil.random("id-"))
+            .key(StringUtil.random("key-"))
+            .displayName(StringUtil.random("display-name-"));
+
+        owner = adminClient.owners().createOwner(owner);
+
+        assertThat(owner)
+            .returns(Owners.SCA_ACCESS_MODE, OwnerDTO::getContentAccessMode)
+            .returns(Owners.ACCESS_MODE_LIST, OwnerDTO::getContentAccessModeList);
+    }
+
+
+    @Test
+    public void shouldAssignTheDefaultModeAndListWhenEmptyStringsAreSpecified() throws Exception {
+        ApiClient adminClient = ApiClients.admin();
+        OwnerDTO owner = new OwnerDTO()
+            .id(StringUtil.random("id-"))
+            .key(StringUtil.random("key-"))
+            .displayName(StringUtil.random("display-name-"))
+            .contentAccessMode("")
+            .contentAccessModeList("");
```

Comment:
It might be better to just use random owner and unset its content access.

Reply:
I agree! Updated here and throughout the other tests